husky_control/launch/control_launch.py

```diff
@@ -1,20 +1,45 @@
 from launch import LaunchDescription
-from ament_index_python.packages import get_package_share_directory
-import launch_ros.actions
-import os
-import yaml
-from launch.substitutions import EnvironmentVariable
-import pathlib
-import launch.actions
-from launch.actions import DeclareLaunchArgument
+from launch.substitutions import PathJoinSubstitution
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
 
 def generate_launch_description():
-    return LaunchDescription([
-        launch_ros.actions.Node(
-            package='robot_localization',
-            executable='ekf_node',
-            name='ekf_node',
-            output='screen',
-            parameters=[os.path.join(get_package_share_directory("husky_control"), 'config', 'localization.yaml')],
-           ),
-])
\ No newline at end of file
+    config_husky_ekf = PathJoinSubstitution(
+        [FindPackageShare("husky_control"),
+        "config",
+        "localization.yaml"],
+    )
+
+    node_ekf = Node(
+        package='robot_localization',
+        executable='ekf_node',
+        name='ekf_node',
+        output='screen',
+        parameters=[config_husky_ekf],
+        )
+
+    config_husky_ekf = PathJoinSubstitution(
```

Comment:
I think this is an accidental duplicate of `config_husky_ekf` above in Lines 8-12